config/crd/bases/tinkerbell.org_hardware.yaml

```diff
@@ -318,6 +341,17 @@ spec:
                     state:
                       type: string
                   type: object
+                resources:
+                  additionalProperties:
+                    anyOf:
+                      - type: integer
```

Comment:
Do we need clarification in the `description` on the assumed unit when specifying as an integer?

Reply:
In-fact I don't think that makes sense because describing a CPUs is a single number. Something still seems odd though, how are users meant to know what value to use?

Reply:
I don't think so, its a Kubernetes type which has [its own documentation](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity)

Reply:
CPUs can be fractional for K8s pods, or whole numbers. See the [Format godoc example](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#example-Format)

Reply:
Right, but fractionality doesn't make sense in Tinkerbell. We only talk in terms of whole CPUs hence I'm wondering if we need to better define boundaries seeming as we're leveraging an existing type with behavior beyond our needs.

Reply:
To clarify, I agree this doesn't need additional documentation on my OP about units assumed with integers. I've shifted to "are our bounds adequately described" because the `resource.Quantity` supports stuff beyond what we want.